apps/web/pages/[user]/calendar-cache/[month].tsx

```diff
@@ -16,7 +16,7 @@ export const getStaticProps: GetStaticProps<
   { user: string }
 > = async (context) => {
   const { user: username, month } = paramsSchema.parse(context.params);
-  const userWithCredentials = await prisma.user.findUnique({
+  const userWithCredentials = await prisma.user.findFirst({
```

Comment:
Should we add org is null to the query? @leog

Reply:
This reminds me that wherever we are querying by username, we need to add that check, otherwise an organisation username might override the settings.

Reply:
Yeah, we need to narrow down the queries now because of orgs. The thing is that we cannot get hostname from GetStaticProps unless we add org as a route parameter as we have [user] for this. Any suggestion @roae?

Reply:
I have faced a similar issue here https://github.com/calcom/cal.com/pull/8993/files#r1229091228

Reply:
This file is addressed by this other PR: https://github.com/calcom/cal.com/pull/9498